### DISCUSSION TITLE
Is DataMesh supporting Data Stewardship for data curation to change data by confirming a data steward?

### DISCUSSION THREAD

**Discussion by alex-datapac** [datamesh-architecture/datamesh-architecture.com#26] [General]
Is DataMesh supporting Data Stewardship for data curation to change data by confirming a data steward?

**Comment by jochenchrist** [comment]
In data mesh, there is no role of a data steward.
The task of curation and management of high-quality data is split up: 
The domain teams are ultimately responsible for the quality of their data under the terms of their data contracts.

Federated Governance can agree on common policies for quality standards.

**Reply by alex-datapac** [reply]
Understand. Is that feature of DataMesh something like Talen's Data Stewardship is offering https://www.talend.com/products/data-stewardship/ ?
